launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -228,6 +233,32 @@ QString CModListView::genModInfoText(CModEntry & mod)
 	if(mod.getValue("contact").isValid())
 		result += urlTemplate.arg(tr("Home")).arg(mod.getValue("contact").toString()).arg(mod.getValue("contact").toString());
 
+	//compatibility info
+	if(mod.isCompatible())
+		result += compatibleString.arg(tr("Compatibility"));
+	else
+	{
+		auto compatibilityInfo = mod.getValue("compatibility").toMap();
+		auto minStr = compatibilityInfo.value("min").toString();
+		auto maxStr = compatibilityInfo.value("max").toString();
+
+		result += incompatibleString.arg(tr("Compatibility"));
+		if(minStr == maxStr)
```

Comment:
We will use this section if both min and max are empty, right?

Reply:
Or is it impossible because we will go if(mod.isCompatible()) way?

Reply:
I see checks on minStr.empty() and maxStr.empty() if both are empty minStr == maxStr. But I expect if both are empty we can not have mod being incompatible because it has no restrictions specified. This is what I want to confirm)

Reply:
Yes, for both empty or other situation if mod is compatible, those checks aren't executed